futurex_openedx_extensions/dashboard/views.py

```diff
@@ -1064,28 +1066,59 @@ def post(self, request: Any, *args: Any, **kwargs: Any) -> Response:
 
 @docs('GlobalRatingView.get')
 class GlobalRatingView(FXViewRoleInfoMixin, APIView):
-    """View to get the global rating"""
+    """View to get the global rating for a single tenant"""
     authentication_classes = default_auth_classes
     permission_classes = [FXHasTenantCourseAccess]
     fx_view_name = 'global_rating'
     fx_default_read_only_roles = ['staff', 'instructor', 'data_researcher', 'org_course_creator_group']
-    fx_view_description = 'api/fx/statistics/v1/rating/: Get the global rating for courses'
+    fx_view_description = 'api/fx/statistics/v1/rating/: Get the global rating for courses in a single tenant'
 
     def get(self, request: Any, *args: Any, **kwargs: Any) -> JsonResponse:
         """
-        GET /api/fx/statistics/v1/rating/?tenant_ids=<tenantIds>
+        GET /api/fx/statistics/v1/rating/?tenant_ids=<tenantId>
 
-        <tenantIds> (optional): a comma-separated list of the tenant IDs to get the information for. If not provided,
-            the API will assume the list of all accessible tenants by the user
+        <tenantId> (required): a single tenant ID to get the rating information for.
+        Multiple tenant IDs are not supported - only one tenant ID must be provided.
         """
-        data_result = get_courses_ratings(fx_permission_info=self.fx_permission_info)
+        tenant_ids_string = request.GET.get('tenant_ids')
+        if not tenant_ids_string:
+            raise FXCodedException(
+                code=FXExceptionCodes.TENANT_ID_REQUIRED_AS_URL_ARG,
+                message='tenant_ids parameter is required'
+            )
+
+        try:
+            tenant_ids = ids_string_to_list(tenant_ids_string)
+        except ValueError as exc:
+            raise FXCodedException(
+                code=FXExceptionCodes.TENANT_NOT_FOUND,
+                message=f'Invalid tenant_ids provided: {str(exc)}'
+            ) from exc
+
+        if len(tenant_ids) != 1:
+            raise FXCodedException(
+                code=FXExceptionCodes.TENANT_ID_REQUIRED_AS_URL_ARG,
+                message=f'Exactly one tenant ID is required, got {len(tenant_ids)}'
+            )
+
+        tenant_id = tenant_ids[0]
```

Comment:
please use the method we already have. And remove any related tests for that expect testing that the view calls `verify_one_tenant_id_provided`
```suggestion
        tenant_id = self.verify_one_tenant_id_provided(request)
```